tools/toolchains/gcc.py

```diff
@@ -79,15 +79,6 @@ def __init__(self, target,  notify=None, macros=None, build_profile=None,
                     self.flags["ld"].append(minimal_printf_wrap)
 
         self.cpu = []
-        if target.is_TrustZone_secure_target:
-            # Enable compiler security extensions
-            self.cpu.append("-mcmse")
-            # Output secure import library
-            self.flags["ld"].extend([
-                "-Wl,--cmse-implib",
-                "-Wl,--out-implib=%s" % join(build_dir, "cmse_lib.o")
```

Comment:
Does TF-M build output a cmse_lib.o file? We need to put that into a place where the NSPE build will pick it up.

Reply:
Yes, but not for PSoC64. Created a ticket to keep track of this requirement https://jira.arm.com/browse/IOTSEC-1266.